pkgs/development/python-modules/msgpack-numpy/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, cython
+, msgpack-python
+, numpy
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "msgpack-numpy";
+  version = "0.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1czf125nixzwskiqiw0145kfj15030sp334cb89gp5w4rz3h7img";
+  };
+
+  buildInputs = [
+    cython
+  ];
+
+  propagatedBuildInputs = [
+   msgpack-python
+   numpy
+  ];
+
+  doCheck = true;
```

Comment:
not needed

Reply:
Why not? There are tests, they work an they take less than 10ms to run?

Reply:
`doCheck` is set to true by default

Reply:
Ah... indeed